scripts/qa_crowbarsetup.sh

```diff
@@ -3577,10 +3587,7 @@ function oncontroller_run_tempest()
     fi
     local imageid=$(glance_image_get_id $image_name)
     crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
-    # test if is cnftools image prepared for tempest
-    wait_for 300 5 \
-        'openstack image show $imageid | grep active &>/dev/null' \
-        "prepare cnftools image"
+    wait_image_active "$image_name" tempest
```

Comment:
The change is intentional. We are getting the imageid value from a image show $image_name so it does not make any difference. And if the name was already found before, we do not upload again, so we do not get duplicate names.